service/ollama_processor.py

```diff
@@ -39,8 +42,12 @@ def can_process(self, model_name: str, model_version: str) -> bool:
         return cached_model
 
     @override
-    def generate_caption(self, model_name: str, model_version: str, image: Image) -> tuple[str, str]:
-        return self._generate_with_prompt(model_name, model_version, [image], caption_prompt)
+    def generate_caption(self, model_name: str, model_version: str, image: Image, prompt) -> tuple[str, str]:
+
+        if prompt == 'default':
```

Comment:
What if `prompt == ''` (empty)?

Reply:
Also, please check / confirm whether the following environment variables can still be used to set a default prompt for the corresponding endpoints:

- `OLLAMA_NSFW_PROMPT`
- `OLLAMA_LABELS_PROMPT`
- `OLLAMA_CAPTION_PROMPT`
  
If they are set to an empty string or "default", the built-in default should be used. This default must never be "default" or empty.